### PR TITLE
feat: display a message for delegates who share their rewards

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/delegates/_example.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ twitter: username
 github: username
 forum: username
 
+# if you share part of the tokens to your supporters 
+rewardsSharing: true | false
```
Comment (dlecan) [inline docs/delegates/_example.md:11]:
```suggestion
rewardsSharing: false
```

File: docs/delegates/_example.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ twitter: username
 github: username
 forum: username
 
+# if you share part of the tokens to your supporters
```
Comment (dlecan) [inline docs/delegates/_example.md:10]:
```suggestion
# if you share part of the tokens to your supporters 
# You should explain below how do you share the tokens
```

**Review by dlecan — CHANGES_REQUESTED** [review]
Arktoshidotcom also shares its rewards I guess

File: docs/delegates/_example.md (hunk 2 above)
Comment (qsvtr) [reply docs/delegates/_example.md:10]: I added a link to the tool that several delegates use

**Comment by qsvtr** [comment]
waiting for cactus's answer, Arktoshidotcom hasn't claimed his profile

File: docs/delegates/6a5418cddb18515517a260bd47fc0e038544e9cc4ac3dac9206523cc366eaa63/README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ github: cactus-15-49
 forum: cactus1549
 email: cactus1549pool@gmail.com
 website: https://marketsquare.io/projects/delegate-cactus1549
+rewardsSharing: true
 ---
```
[the remark this answers is not in the document]
Comment (Cactus-15-49) [reply docs/delegates/6a5418cddb18515517a260bd47fc0e038544e9cc4ac3dac9206523cc366eaa63/README.md:8]: no, I don't share for now

**Comment by qsvtr** [comment]
it's done @dlecan

**Review by dlecan — APPROVED** [review]

**Pull request merged**